bears/general/ImageDimensionBear.py

```diff
@@ -0,0 +1,87 @@
+from glob import glob
+import os
+import yaml
+import shutil
+
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from dependency_management.requirements.GemRequirement import GemRequirement
+from dependency_management.requirements.PipRequirement import PipRequirement
+from sarge import run, Capture
+
+
+class ImageDimensionBear(GlobalBear):
+    """
+    Checks the dimension of an image us a gem.
+
+    More information is available at <github.com/Abhi2424shek/img_checker>
+    """
+
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    REQUIREMENTS = {GemRequirement('img_checker'),
+                    PipRequirement('pyyaml', '3.12')}
+    LICENSE = 'AGPL-3.0'
+    CAN_FIX = {'Image Dimension'}
+
+    @classmethod
+    def check_prerequisites(cls):
+        if shutil.which('img_checker') is None:
+            return 'img_checker is not installed.'
+        return True
+
+    def run(self,
+            image_file: str,
+            width: int,
+            height: int):
+        """
+        This bear ensures that images used in your project are
+        within a fixed dimension.
+
+        WARNING: If your repository has an existing img_config.yml
+        file, please change it since this bear creates a file
+        with that name and removes it in the end.
+
+        :param image_file: The file/directory that bear
+                           will look for images
+        :param width:      The maximum width value allowed
+                           for images
+        :param height:     The maximum height value allowed
+                           for images
+        """
+
+        warning_message = ('If there is any img_config.yml file'
+                           ' in your root directory, this bear'
+                           ' will remove it')
+
+        self.warn(warning_message)
+
+        if (glob(image_file) == []):
```

Comment:
These validators should be separate functions.  See `known_checkstyle_or_path` in `CheckstyleBear`.